modules/navier_stokes/test/tests/finite_volume/ins/solidification/tests

```diff
@@ -0,0 +1,40 @@
+[Tests]
+  design = 'NSComputeLiquidFraction.md NSFVPhaseChangeSource.md NSMixtureMaterial.md'
+  issues = '#23357'
+  [solidification_exo_no_advection]
+    type = 'Exodiff'
+    input = 'solidification_no_advection.i'
+    exodiff = 'solidification_no_advection_out.e'
+    requirement = 'The system shall be able to solve solidification without advection using the finite volume method.'
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+  []
+  [solidification_exo_pipe]
+    type = 'Exodiff'
+    input = 'pipe_solidification.i'
+    exodiff = 'pipe_solidification_out.e'
+    requirement = 'The system shall be able to solve solidification with advection in a pipe using the finite volume method.'
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+  []
+  [solidification_enthalpy_balance]
+    type = 'CSVDiff'
+    input = 'pipe_solidification.i'
+    csvdiff = 'pipe_solidification_csv.csv'
+    requirement = 'The system shall be able to conserve energy during solidification.'
+    rel_err = 1.0E-5
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+    recover = false
```

Comment:
why do you need that?

Reply:
We would only like to print CSV on FINAL. If we don't have a dedicated csv output it doesn't print anything. We get this if we print the csv at the end with recover:
```
time,average_T
10000,292.78702201863
time,average_T
10000,292.78702201863
time,average_T
10000,292.78702201863
```
it looks like it keeps appending to the same file instead of clearing it. The vector postprocessor seemingly works